client/index.html

```diff
@@ -169,143 +169,126 @@ <h1 class="title">Connect</h1>
-										<label for="new_password_input" class="sr-only">Enter desired new password</label>
-										<input type="password" id="new_password_input" name="new_password" class="input" placeholder="Enter desired new password">
+						</table>
+						
+						<table class="settings">
+							<tr>
+								<th class="section-title">Notifications</th>
+								<th class="icon"></th>
+								<th class="icon"></th>
+							</tr>
+							<tr>
+								<td>
+									<label for="setting:desktopNotifications">Enable desktop notifications</label>
+									<div class="error" id="warnDisabledDesktopNotifications">
+										<strong>Warning</strong>: Desktop notifications are blocked by your web browser
 									</div>
-									<div class="col-sm-12">
-										<label for="verify_password_input" class="sr-only">Repeat new password</label>
-										<input type="password" id="verify_password_input" name="verify_password" class="input" placeholder="Repeat new password">
-									</div>
-									<div class="col-sm-12 feedback"></div>
-									<div class="col-sm-12">
-										<button type="submit" class="btn">Change password</button>
-									</div>
-								</form>
-							</div>
-							<% } %>
-							<div class="col-sm-12">
-								<h2>Custom Stylesheet</h2>
-							</div>
-							<div class="col-sm-12">
-								<textarea class="input" name="userStyles" id="user-specified-css-input" placeholder="You can override any style with CSS here"></textarea>
-							</div>
-							<div class="col-sm-12">
-								<h2>About The Lounge</h2>
-							</div>
-							<div class="col-sm-12">
-								<p class="about">
-									You're currently running version <%= version %><br>
-									<a href="https://github.com/thelounge/lounge/blob/master/CHANGELOG.md#readme" target="_blank">View the change log</a>
-								</p>
-							</div>
+								</td>
+								<td> <input type="checkbox" id="setting:desktopNotifications" name="setting:desktopNotifications"> </td>
+								<td> <input type="checkbox" id="sync:desktopNotifications"    name="sync:desktopNotifications"> </td>
+							</tr>
+							<tr>
+								<td>
+									<label for="setting:notification">Enable notification sound</label>
+									<button id="play">(<span class="icon"></span> play sound)</button>
+								</td>
+								<td> <input type="checkbox" id="setting:notification" name="setting:notification"> </td>
+								<td> <input type="checkbox" id="sync:notification"    name="sync:notification"> </td>
+							</tr>
+							<tr>
+								<td> <label for="setting:notifyAllMessages">Enable notification for all messages</label> </td>
+								<td> <input type="checkbox" id="setting:notifyAllMessages" name="setting:notifyAllMessages"> </td>
+								<td> <input type="checkbox" id="sync:notifyAllMessages"    name="sync:notifyAllMessages"> </td>
+							</tr>
+						</table>
+						
+						<div class="col-sm-12">
+							<h2>
+								Custom Stylesheet
+								<% if (!public) { %>
```

Comment:
Only this sync checkbox is hidden in public mode, that's not quite consistent. Could we just set `disabled` on all the sync checkboxes, and perhaps even add a tooltip saying syncing is not allowed in public?